backends/rtlil/rtlil_backend.cc

```diff
@@ -115,13 +115,17 @@ void RTLIL_BACKEND::dump_sigspec(std::ostream &f, const RTLIL::SigSpec &sig, boo
 	}
 }
 
-void RTLIL_BACKEND::dump_wire(std::ostream &f, std::string indent, const RTLIL::Wire *wire)
+void RTLIL_BACKEND::dump_wire(std::ostream &f, std::string indent, const RTLIL::Wire *wire, bool flag_d)
 {
 	for (auto &it : wire->attributes) {
 		f << stringf("%s" "attribute %s ", indent.c_str(), it.first.c_str());
 		dump_const(f, it.second);
 		f << stringf("\n");
 	}
+	if (flag_d && wire->driverCell) {
+		f << stringf("%s" "driver %s %s\n", indent.c_str(),
+				wire->driverCell->name.c_str(), wire->driverPort.c_str());
```

Comment:
Since it should be always possible to reconstruct this data, why not dump it as a comment in the RTLIL output? (Does RTLIL support comments?) And if we do dump it as a comment, maybe we could make it non-optional and drop the argument to `dump_wire`.

Reply:
...and drop the other new dump arguments and the option to the pass.